src/CMakeLists.txt

```diff
@@ -56,7 +56,9 @@ endif()
 
 # The option is forced to ON if _GLIBCXX_USE_CXX11_ABI=0
 cmake_dependent_option(BUILD_FAST_TOKENIZERS "Compile core_tokenizers instead of downloading prebuilt library" OFF "NOT USE_ABI0" ON)
-
+if(NOT BUILD_FAST_TOKENIZERS AND NOT CMAKE_BUILD_TYPE MATCHES Release)
```

Comment:
For multi config generator cmake build type is usually not defined (or at least it's wrong to define it)

We need to use an approach similar to https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/pyopenvino/CMakeLists.txt#L74-L88

Reply:
changed